src/meta/src/rpc/ddl_controller.rs

```diff
@@ -873,13 +892,25 @@ impl DdlController {
                                             format!("ProjectExecutor(from sink {})", sink_id);
                                     }
 
-                                    **merge_node = MergeNode {
-                                        upstream_actor_id: sink_actor_ids.clone(),
-                                        upstream_fragment_id,
-                                        upstream_dispatcher_type: DispatcherType::Hash as _,
-                                        fields: sink_fields.to_vec(),
+                                    **merge_node = {
+                                        #[expect(deprecated)]
```

Comment:
Is it possible to move the attributes onto the field?

Reply:
Just had a try, but the attribute doesn't work.